chrome_extension/vendor/mooltipass/mcCombinations.js

```diff
@@ -746,9 +746,7 @@ mcCombinations.prototype.detectCombination = function() {
 							.unbind('click.mooltipass')
 							.on('click.mooltipass', this.onSubmit.bind(this, { target: currentForm.element[0] }))
 							
-						mpJQ()
-							.add(currentForm.combination.fields.username)
-							.add(currentForm.combination.fields.password)
+						mpJQ(mpJQ.map(currentForm.fields, function (field) { return field.get() }))
```

Comment:
can you explain what his does?

Reply:
@limpkin 
Instead of listening for return key on username and password fields, we listen on all fields. `currentForm.fields` is an array of jQuery objects and this line transforms this array to jQuery object with multiple elements.

Reply:
@limpkin https://stackoverflow.com/questions/11304173/how-to-create-jquery-object-from-array-of-jquery-objects

Reply:
👍